File: JavaScript/Array/L_threeConseqOdd.js
var threeConsecutiveOdds = function (arr) {
    function is_odd(num) {
        return num % 2;
    }
    for (let i = 0; i < arr.length - 2; i++) {
        if (is_odd(arr[i]) && is_odd(arr[i + 1]) && is_odd(arr[i + 2])) return true;
    }
    return false;
};

File: Python/Arrays/L_threeConseqOdd.py
def threeConsecutiveOdds(arr) -> bool:
    def is_odd(n):
        return n % 2
    for i in range(len(arr) - 2):
        if is_odd(arr[i]) and is_odd(arr[i+1]) and is_odd(arr[i+2]):
            return True
    return False